### DISCUSSION TITLE
File Structure: Bind item configurations and textures together

### DISCUSSION THREAD

**Discussion by TheFloodDragon** [oraxen/oraxen#1467] [Ideas]
Could we have a structure that like ItemsAdder, has bound item configs and resource packs to classify custom texture items?

They may called groups which include config and resource pack together.

Hope's for your reply, thanks

**Comment by Boy0000** [comment]
not planning on this type of structure no
would require a rework of the entire plugin for no real benefit in my eyes

**Reply by TheFloodDragon** [reply]
well.
